docs/extending/developing-code/calling-mathics-library-from-within-python-code.rst

```diff
@@ -94,16 +94,16 @@ Although we set ``add_builtin`` explicitly for pedagodical purpose,
 leave it off in future examples.
 
 
-Mathics Results as Python Objects
----------------------------------
+Mathics3 Results as Python Objects
+----------------------------------
```

Comment:
Is there a reason for using `------` here? (other places use `=====`)

Reply:
The `-` and `=` are different section levels